constraint/config/crds/templates.gatekeeper.sh_constrainttemplates.yaml

```diff
@@ -147,6 +141,8 @@ spec:
                             type: array
                         type: object
                       validation:
+                        default:
+                          legacySchema: true
```

Comment:
Thanks for working on this PR @maxsmythe! With this change, users would still need to update their CRDs right?

Reply:
You're welcome! :)

Users still need to update their CRDs. In general, it seems dangerous to allow CRDs to go stale.

I talk a bit more about the CRD thing in Oren's issue:

https://github.com/open-policy-agent/gatekeeper/issues/1458

Basically saying that we could paper over having an old CRD, but that would eventually break anyway once we moved to defaulting to v1 CRDs.

Reply:
Also, if we do want to paper over having an out-of-date CRD manifest, we'd have to do that in the Gatekeeper repo.